build-n-release-w-docker.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+
+pkgName="doppler-webapp"
+cdnBaseUrl=${2:-"//cdn.fromdoppler.com/$pkgName"}
+pkgVersionQaPrefix="qa-"
+pkgVersionIntPrefix="int-"
+pkgVersionDevelopmentPrefix="dev-"
+
+# Stop script on NZEC
+set -e
+# Stop script if unbound variable found (use ${var:-} if intentional)
+set -u
+
+# Lines added to get the script running in the script path shell context
+# reference: http://www.ostricher.com/2014/10/the-right-way-to-get-the-directory-of-a-bash-script/
+cd $(dirname $0)
+
+# To avoid issues with MINGW y Git Bash, see:
+# https://github.com/docker/toolbox/issues/673
+# https://gist.github.com/borekb/cb1536a3685ca6fc0ad9a028e6a959e3
+export MSYS_NO_PATHCONV=1
+export MSYS2_ARG_CONV_EXCL="*"
+
+# build production package and generate version number
+docker run --rm \
+    -v `pwd`:/work \
+    -w /work \
+    node:10 \
+    /bin/sh -c "\
+        yarn \
+        && yarn semantic-release \
+    "
+
+# read pkgVersion from version.txt (see package.json => $.release.prepare[?(@.path=="@semantic-release/exec")])
+pkgVersion=v$(cat build/version.txt)
+
+# TODO: generate docker image here
+
+echo Publishing to Akamai...
+echo pkgName: $pkgName
+echo pkgVersion: $pkgVersion
+echo cdnBaseUrl: $cdnBaseUrl
+
+# Force pull the latest image version due to the cache not always is pruned immediately after an update is uploaded to docker hub
+docker pull dopplerrelay/doppler-relay-akamai-publish
+
+docker run --rm \
```

Comment:
This is production build right?

Reply:
Mmm, it depends on what is "production" for you.

It copies the result files of production build (the result of `...&& yarn semantic-release`) to our CDN.

Does it answer your question?

Reply:
So it is production then 👍